clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/download/getE4EMock.utility.ts

```diff
@@ -0,0 +1,177 @@
+    ["copybook", "fingerprint"],
+    ["copybook2", "fingerprint2"],
+  ]),
+  listMembers: jest.fn().mockReturnValue([
+    {
+      dataset: "dataset",
+      member: "copybook",
+    },
+  ]),
+  onDidChangeElement: jest.fn(),
+  getElement: jest.fn().mockReturnValue(["content", "fingerprint"]),
+  getMember: jest.fn().mockReturnValue("content"),
+};
+
+export const e4eErrorMock: E4E = {
+  getConfiguration: jest.fn().mockReturnValue({
+    pgms: {
+      program: "program",
+      pgroup: "pgroup",
+    },
+    pgroups: [
+      {
+        name: "string",
+        libs: [
+          { dataset: "dataset" },
+          {
+            environment: "environment",
+            stage: "stage",
+            system: "system",
+            subsystem: "subsystem",
+            type: "type",
+            use_map: false,
+          },
+        ],
+      },
+    ],
+  }),
+  getProfileInfo: jest.fn().mockReturnValue({
+    profile: "profile",
+    instance: "string",
+  }),
+  isEndevorElement: jest.fn().mockReturnValue(true),
+  listElements: jest.fn().mockReturnValue(["filename", "fingerprint"]),
+  listMembers: jest.fn().mockReturnValue(["member"]),
+  onDidChangeElement: jest.fn(),
+  getElement: jest.fn().mockReturnValue(new Error("error")),
+  getMember: jest.fn().mockReturnValue(new Error("error")),
+};
+
+export const e4eResponseMock: e4eResponse = {
+  configuration: {
+    name: "pgroup",
+    libs: [
+      { dataset: "dataset" },
+      {
+        environment: "environment",
+        stage: "stage",
+        system: "system",
+        subsystem: "subsystem",
+        type: "type",
+        use_map: false,
+      },
+    ],
+  },
+  api: e4eMock,
+  profile: { instance: "instance", profile: "profile" },
+  uri: "document-uri",
+  elements: [
+    {
+      use_map: false,
+      environment: "environment",
+      stage: "stage",
+      system: "system",
+      subsystem: "subsystem",
+      type: "type",
+      element: "copybook",
+      fingerprint: "fingerprint",
+    },
+  ],
+  members: [
+    {
+      dataset: "dataset",
+      member: "copybook",
+    },
+  ],
+};
+
+export const e4eErrorResponseMock: e4eResponse = {
+  configuration: {
+    name: "pgroup",
+    libs: [
+      { dataset: "dataset" },
+      {
+        environment: "environment",
+        stage: "stage",
+        system: "system",
+        subsystem: "subsystem",
+        type: "type",
+        use_map: false,
+      },
+    ],
+  },
+  api: e4eErrorMock,
+  profile: { instance: "instance", profile: "profile" },
+  uri: "document-uri",
+  elements: [
+    {
+      use_map: false,
+      environment: "environment",
+      stage: "stage",
+      system: "system",
+      subsystem: "subsystem",
+      type: "type",
+      element: "copybook",
+      fingerprint: "fingerprint",
+    },
+  ],
+  members: [
+    {
+      dataset: "dataset",
+      member: "copybook",
+    },
+  ],
+};
+
+it("dummy", () => {
+  expect(true).toBeTruthy();
+});
```

Comment:
why is this here?

Reply:
It is placed under tests. so they should contain at least 1 case

Reply:
Why is it here at all? As far as I can tell, it is not executed and it actually does not do anything useful.

Reply:
I think the `.utility.ts` extension is not excluded by the list in jest.config.js. Either change the filename to remove the `utility` part, or adjust the jest.config.js to exclude this patter.